iib/web/static/api_v1.yaml

```diff
@@ -0,0 +1,461 @@
+        state_reason:
+          type: string
+          example: Completed successfully
+        updated:
+          type: string
+          example: '2019-09-17T19:42:47.149979'
+        user:
+          type: string
+          example: msdhoni@DOMAIN.LOCAL
+    RequestAdd:
+      type: object
+      properties:
+        add_arches:
+          type: array
+          items:
+            type: string
+          example:
+            - amd64
+            - s390x
+        binary_image:
+          type: string
+          example: 'quay.io/operator-framework/upstream-registry-builder:v1.5.9'
+        bundles:
+          type: array
+          items:
+            type: string
+          example:
+            - 'quay.io/user/openshift-bundle-example-container:v1'
+        cnr_token:
+          type: string
+          description: >
+            This token is required only if support for legacy app registry is
+            required.
+          example: token
+        from_index:
+          type: string
+          example: 'quay.io/iib-stage/iib:4'
+        organization:
+          type: string
+          description: >
+            The organization is required only if support for legacy app registry
+            is required.
+          example: openshift-operators
+      required:
+        - binary_image
+        - bundles
+    RequestRemove:
+      type: object
+      properties:
+        add_arches:
+          type: array
+          items:
+            type: string
+          example:
+            - amd64
+            - s390x
+        binary_image:
+          type: string
+          example: 'quay.io/operator-framework/upstream-registry-builder:v1.5.9'
+        from_index:
+          type: string
+          example: 'quay.io/iib-stage/iib:4'
+        operators:
+          type: array
+          items:
+            type: string
+          example:
+            - quay.io/user/openshift-operator
+      required:
+        - binary_image
+        - from_index
+        - operators
+    RequestUpdate:
+      type: object
+      properties:
+        arches:
+          type: array
+          items:
+            type: string
+          example:
+            - amd64
+            - s390x
+        binary_image_resolved:
+          type: string
+          example: >-
+            quay.io/operator-framework/upstream-registry-builder@sha256:7d8e5dddad0275bc903b6ef17840f98441b15b8b999609af2c9579960e52080e
+        bundle_mapping:
+          $ref: '#/components/schemas/BundleMapping'
+        from_index_resolved:
+          type: string
+          example: >-
+            quay.io/iib-stage/iib@sha256:7d8e5dddad0275bc903b6ef17840f98441b15b8b999609af2c9579960e52080e
+        index_image:
+          type: string
+          example: 'quay.io/iib-stage/iib:4'
+        state:
+          type: string
+          example: complete
+        state_reason:
+          type: string
+          example: Completed successfully
+    RequestVerbose:
+      allOf:
+        - type: object
```

Comment:
Could you swap the order of the array so that `state_history` is at the bottom? This looks better since it lists all the properties in a non-verbose request first and then shows the additional properties.